### PR TITLE
chore: prepare for 8.26.0

### PR DESCRIPTION
**Pull request by dariakoko**
This PR resolves [] 

### PR Checklist

- [ ] I have tested my changes on **iOS** and **Android**.
- [ ] I hid my changes behind a **[feature flag]**, or they don't need one.
- [ ] I have included **screenshots** or **videos**, or I have not changed the UI.
- [ ] I have added **tests**, or my changes don't require any.
- [ ] I added an **[app state migration]**, or my changes do not require one.
- [ ] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [ ] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

#nochangelog

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

**Review by gkartalis — COMMENTED** [review]
Can we get rid of the changes on the `android/app/build.gradle` file before merging?

File: android/app/build.gradle [hunk 1]
```diff
@@ -147,8 +147,8 @@ android {
         applicationId "net.artsy.app"
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
-        versionCode 7
-        versionName "1.0.0"
+        versionCode 1674645499
+        versionName "8.25.0"
```
Comment (dariakoko) [inline android/app/build.gradle:151]: This change looks somehow wrong. Do you know why the old version code and name are like this?

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  ✅ **No changelog changes**
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against b1dba0a1098a81bb9ea2dcc3b628bd92f537bbc8
</p>

**Comment by dariakoko** [comment]
@gkartalis should we update the script so it doesn't touch `build.gradle` at all?

**Comment by gkartalis** [comment]
@dariakoko we can look into improving this for sure.

The `versionCode` and `versionName` in the `android/app/build.gradle` is essential before each release from PlayStore for version tracking.

The reason that it looks a bit weird is because fastlane needs some things there to automate the `versionCode` and `versionName` bump.

Happy to talk it through more if you want - Added a card on the #practice-board [here](https://www.notion.so/artsy/Releases-Look-into-reverting-the-android-app-build-gradle-diff-after-we-submit-to-stores-ffe34a95a5d843ed80eb9135f8da246d?pvs=4)

**Pull request merged**